File: back-end/src/main/java/vn/com/cardoctor/garage_service/entities/InboundProduct.java
package vn.com.cardoctor.garage_service.entities;

import lombok.Data;

import javax.persistence.*;
import java.math.BigDecimal;

@Entity
@Data
@Table(name = "inbound_products")
public class InboundProduct extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private Long inboundTicketId;
    private Long productId;
    private String unit;
    private Long distributorId;
    private BigDecimal requestQuantity;
    private BigDecimal exportQuantity;
    private String note;
    private Integer status;
}

File: front-end/src/types/InboundProduct.ts
type InboundProduct = {
	id: number
	productId: number | null
	inboundTicketId: number
	unit?: string
	distributorId?: number
	requestQuantity?: number
	exportQuantity?: number
	note?: string
	status: number
	instanceKey: number
	productName?: string
}
